tests/templates/kuttl/tls/10-assert.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+metadata:
+  name: install-trino
```

Comment:
```suggestion
```

Reply:
These don't seem to do anything anyway?

Reply:
Its the name of that step?

Reply:
I don't see it mentioned in https://kuttl.dev/docs/testing/reference.html#testassert, are you sure this is actually used by it?

Reply:
removed